webpack.common.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   output: {
     publicPath: '/',
     path: path.resolve(__dirname, 'dist'),
-    filename: 'app.js',
+    filename: 'app.[hash].js',
   },
```

Comment:
too easy :)

Reply:
have to check config options for this...

Reply:
I was trying to be funny. If this is all it takes, very cool!

Reply:
yeah, what I wanted to say is that this is something that we would probably need to change when doing code splitting, and TBH I haven't spend much time finding out exactly the options we had here. Just went for the straightforward solution, now.